### PR TITLE
CCE and CIS numbers assignments for SLE 12/15 rules which belong to u…

### PR DESCRIPTION
**Pull request by rumch-se**
…ser and group settings

#### Description:

- _ Small updates related to SLE 12/15 rules which are part of CIS group 6.2 User and Group Settings _

#### Rationale:

- Some of the SLE 12/15 rules do not have CCE/CIS numbers assigned or CIS numbers are not correct  

#### Review Hints:

- _Review hints here. Replace this text. Don't use the italics format!_

- _Use this optional section to give any relevant information which could help the reviewer to more quickly and assertively understand and test the changes._

- _Good examples are useful commands, if it is better to review all commits together or in a suggested sequence, any relevant discussion in other PRs or issues, etc._

### CODE REVIEW

File: shared/references/cce-sle12-avail.txt [hunk 1]
```diff
@@ -1,72 +1,3 @@
-CCE-92227-8
```
Comment (marcusburghardt) [inline shared/references/cce-sle12-avail.txt:1]: It seems more CCEs were removed from this list than included in the rule.yml files in this PR. Is that intentional?

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rumen

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rume

File: shared/references/cce-sle12-avail.txt [hunk 2]
```diff
@@ -1,46 +1,3 @@
...
-CCE-92287-2
-CCE-92288-0
-CCE-92289-8
-CCE-92290-6
-CCE-92291-4
-CCE-92292-2
-CCE-92293-0
-CCE-92294-8
-CCE-92295-5
-CCE-92296-3
```
Comment (marcusburghardt) [inline shared/references/cce-sle12-avail.txt:43]: You are probably removing more CCEs than necessary in this PR. Could you check, please?

File: shared/references/cce-sle15-avail.txt [hunk 3]
```diff
@@ -1,39 +1,3 @@
...
-CCE-91400-2
-CCE-91401-0
-CCE-91402-8
-CCE-91403-6
-CCE-91404-4
-CCE-91405-1
-CCE-91406-9
-CCE-91407-7
-CCE-91408-5
-CCE-91409-3
```
Comment (marcusburghardt) [inline shared/references/cce-sle15-avail.txt:36]: You are probably removing more CCEs than necessary in this PR. Could you check, please?

**Review by marcusburghardt — CHANGES_REQUESTED** [review]

**Comment by rumch-se** [comment]
Hello @marcusburghardt 
I have done expected changes. I do believe that now everything is OK.
Have a nice day
Rumen

**Review by marcusburghardt — APPROVED** [review]

**Pull request merged**